spec/contivModel2raml.rb

```diff
@@ -32,8 +37,8 @@
 
       if p["type"] == "array"
         p["items"] = {
-		"type" => value["items"]
```

Comment:
does this take out items node from underneath array. I think raml spec needs it (or raml2html tool does to generate correct html). So
```
foo:
  type: array
  items:
      type: bar
```
is needed.